requirements/edx/github.in

```diff
@@ -73,5 +73,5 @@ git+https://github.com/edx/django-require.git@0c54adb167142383b26ea6b3edecc32118
 
 # Third Party XBlocks
 
-git+https://github.com/open-craft/xblock-poll@922cd36fb1c3cfe00b4ce03b19a13185d136447d#egg=xblock-poll==1.10.2
+git+https://github.com/open-craft/xblock-poll@v1.12.0#egg=xblock-poll==1.12.0
```

Comment:
Maybe I'm being picky, but could this be in a separate PR since it seems unrelated. Feel free to ping me on it if you need a quick review.

Reply:
Thanks. I wasn't clear this was related.